gwpy/table/tests/test_table.py

```diff
@@ -759,6 +759,19 @@ def test_read_snax(self):
                 filter_table(table, 'snr>.5')[('time', 'snr')],
             )
 
+            # check error handling when missing channels are encountered
+            missing = [channel, 'H1:MISSING']
+            with pytest.raises(ValueError):
+                self.TABLE.read(fp, channels=missing, format='hdf5.snax')
+
+            with pytest.warns(None):
```

Comment:
Why is `None` being used here, doesn't that function require a Warning type. I think this should be `UserWarning`
```suggestion
            with pytest.warns(UserWarning):
```

Reply:
Thanks @duncanmmacleod, I just updated that. I originally went with `None` since I wasn't sure that `warnings.warn()` without specifying a specific warning always defaulted to a `UserWarning` across python versions. Putting `None` is a catch all for catching any warnings raised: https://docs.pytest.org/en/3.0.5/recwarn.html#recwarn.